src/safeds/data/tabular/containers/_time_series.py

```diff
@@ -118,21 +116,38 @@ def _from_table_to_time_series(
         Examples
         --------
         >>> from safeds.data.tabular.containers import Table, TimeSeries
-        >>> table = Table({"date": ["01.01", "01.02", "01.03", "01.04"], "f1": ["a", "b", "c", "a"], "t": [1,2,3,4]})
-        >>> timeseries = TimeSeries._from_table_to_time_series(table, "t", "date", ["f1"])
+        >>> test_table = Table({"date": ["01.01", "01.02", "01.03", "01.04"], "f1": ["a", "b", "c", "a"], "t": [1,2,3,4]})
+        >>> timeseries = TimeSeries._from_table(test_table, "t", "date", ["f1"])
         """
         if feature_names is not None and time_name in feature_names:
             raise ValueError(f"Column '{time_name}' can not be time and feature column.")
+        if feature_names is not None and target_name in feature_names:
+            raise ValueError(f"Column '{target_name}' can not be target and feature column.")
+
+        if target_name not in table.column_names:
+            raise UnknownColumnNameError([target_name])
 
+        result = object.__new__(TimeSeries)
+
+        result._data = table._data
+        result._schema = table._schema
+        result._time = table.get_column(time_name)
+        result._target = table.get_column(target_name)
         if feature_names is None:
-            feature_names = table.column_names
-            if time_name in feature_names:
-                feature_names.remove(time_name)
-            if target_name in feature_names:
-                feature_names.remove(target_name)
-        tagged_table = TaggedTable._from_table(table=table, target_name=target_name, feature_names=feature_names)
+            result._feature_names = []
+            feature_names = []
+            result._features = Table()
+        if len(feature_names) == 0:
+            result._feature_names = []
+            result._features = Table()
+        else:
+            result._feature_names = feature_names
+            # for some reason it gets converted into TimeSeries sometimes
```

Comment:
1. What do you mean by "it"? 
2. Why does this happen? 
3. Is this an issue?

Reply:
[Template method](https://refactoring.guru/design-patterns/template-method/python/example).

But since a `TimeSeries` is a `Table` by your definition, it should be perfectly OK to pass a `TimeSeries` where a `Table` is expected. Thus, I'd suggest removing this comment.

Reply:
Yes, I removed the comment and added the ```as_table```-funvtion to the beginning of the function, so it is similar to the ```TaggedTable```